src/main/java/com/jagrosh/jmusicbot/utils/TimeUtil.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.utils;
+
+import java.util.regex.Pattern;
+
+public class TimeUtil
+{
+
+    public static String formatTime(long duration)
+    {
+        if(duration == Long.MAX_VALUE)
+            return "LIVE";
+        long seconds = Math.round(duration/1000.0);
+        long hours = seconds/(60*60);
+        seconds %= 60*60;
+        long minutes = seconds/60;
+        seconds %= 60;
+        return (hours>0 ? hours+":" : "") + (minutes<10 ? "0"+minutes : minutes) + ":" + (seconds<10 ? "0"+seconds : seconds);
+    }
+
+    /**
+     * @param args formatted as: [+ | -] &lt;HH:MM:SS | MM:SS | SS&gt;
+     * @return Time in milliseconds
+     */
+    public static SeekTime parseTime(String args)
+    {
+        Boolean seek_relative = null; // seek forward or backward
+        char charRelative = args.charAt(0);
+        if (args.charAt(0) == '+' || args.charAt(0) == '-')
+        {
+            args = args.substring(1);
+            seek_relative = charRelative == '+';
+        }
```

Comment:
I feel like this can be simplified to:
```java
if(args.length() == 0)
    return null;
boolean relative = false;
char first = args.charAt(0);
if(first == '+' || first == '-')
{
    args = args.substring(1);
    relative = true;
}
```
(also note that I changed `seek_relative` to `relative`; `seekRelative` would have been fine too, but don't use snake_case for variable names)

Reply:
Whoops. Haven't worked with Java in a while, guess it's showing :P

Reply:
This assumes we will always seek relatively though.

Reply:
Also I missed this in my earlier review but don't forget to make the final millisecond value negative if `first == '-'`

Reply:
> This assumes we will always seek relatively though.

No, we only set relative to true if it starts with `+` or `-`. By default, `relative` will be `false` and so we won't seek relative.